src/plugins/polyfillSettings.ts

```diff
@@ -0,0 +1,12 @@
+import companionTranslations from './companionTranslations';
+import gettextFactoryConst from './gettextFactory.const';
+import { i18nModuleString } from './i18n';
+import polyfill from './polyfill';
+
+export default function polyfillCompanion() {
+  return polyfill({
+    '\0lang:table': companionTranslations('settings/**/*.po'),
```

Comment:
Should these null prefixed keys b econstants exported from `i18n.ts` maybe?

Reply:
That, or change `i18n.ts` to export a function which returns the necessary polyfills. E.g.:
```ts
export default function i18nPolyfill(translationsGlob: string) {
  return {
    '\0lang:table': companionTranslations(translationsGlob),
    '\0lang:gettext-factory': gettextFactoryConst,
    i18n: i18nModuleString,
  };
}
```